unittests/forked_tests.cpp

```diff
@@ -118,14 +124,15 @@ BOOST_AUTO_TEST_CASE( fork_with_bad_block ) try {
    }
 
    // make sure we can still produce a blocks until irreversibility moves
-   auto lib = bios.control->head_block_state_legacy()->dpos_irreversible_blocknum;
+   auto lib = bios.lib_block->block_num();
```

Comment:
Because the lib is now immediately available in the tester (updated from the ` irreversible_block` signal) I thought it made more sense to use it rather than quering the controller.